tests/core.py

```diff
@@ -1188,33 +1218,13 @@ def test_cli_connections_list(self):
 
         # Assert that some of the connections are present in the output as
         # expected:
-        self.assertIn(['aws_default', 'aws'], conns)
-        self.assertIn(['beeline_default', 'beeline'], conns)
-        self.assertIn(['emr_default', 'emr'], conns)
-        self.assertIn(['mssql_default', 'mssql'], conns)
-        self.assertIn(['mysql_default', 'mysql'], conns)
-        self.assertIn(['postgres_default', 'postgres'], conns)
-        self.assertIn(['wasb_default', 'wasb'], conns)
-        self.assertIn(['segment_default', 'segment'], conns)
-
-        # Attempt to list connections with invalid cli args
-        with mock.patch('sys.stdout',
-                        new_callable=six.StringIO) as mock_stdout:
-            cli.connections(self.parser.parse_args(
-                ['connections', '--list', '--conn_id=fake', '--conn_uri=fake-uri',
-                 '--conn_type=fake-type', '--conn_host=fake_host',
-                 '--conn_login=fake_login', '--conn_password=fake_password',
-                 '--conn_schema=fake_schema', '--conn_port=fake_port', '--conn_extra=fake_extra']))
-            stdout = mock_stdout.getvalue()
-
-        # Check list attempt stdout
-        lines = [l for l in stdout.split('\n') if len(l) > 0]
-        self.assertListEqual(lines, [
-            ("\tThe following args are not compatible with the " +
```

Comment:
What has happened to this test case? I think, it would still be useful, but I don't see it anymore?

Reply:
This test case was removed because the behavior of rejecting arguments for the list command was removed. It just lists the connections no matter what args are provided now.